cpp/oneapi/dal/algo/covariance/detail/compute_ops.cpp

```diff
@@ -23,12 +24,36 @@ namespace v1 {
 
 template <typename Policy, typename Float, typename Method, typename Task>
 struct compute_ops_dispatcher<Policy, Float, Method, Task> {
-    compute_result<Task> operator()(const Policy& policy,
+    compute_result<Task> operator()(const Policy& ctx,
```

Comment:
Why are you renaming the `policy` to `ctx`? I see both names make sense, but for most algos we seem to be using `Policy& policy` (even though `Policy& ctx` is used frequently, too). Can't we just pick one and be consistent? Or is there a deeper meaning in the name?